packages/sui-lint/eslintrc.ts.js

```diff
@@ -199,11 +199,20 @@ module.exports = {
     'prefer-regex-literals': RULES.WARNING,
     'prettier/prettier': [RULES.ERROR, prettierOptions],
     'simple-import-sort/imports': [RULES.WARNING, {groups: IMPORT_SORT_GROUPS}],
-    'react/jsx-no-bind': RULES.OFF
-    // 'sui/factory-pattern': RULES.WARNING,
-    // 'sui/serialize-deserialize': RULES.WARNING
+    'react/jsx-no-bind': RULES.OFF,
+    'sui/commonjs': RULES.WARNING,
+    'sui/layersArch': RULES.WARNING
```

Comment:
I would use param-case for the rules naming...
```suggestion
    'sui/layers-arch': RULES.WARNING
```